renderdoc/driver/d3d12/d3d12_commands.cpp

```diff
@@ -270,7 +270,7 @@ ULONG STDMETHODCALLTYPE WrappedID3D12DebugCommandQueue::AddRef()
 
 ULONG STDMETHODCALLTYPE WrappedID3D12DebugCommandQueue::Release()
 {
-  m_pQueue->Release();
+  SAFE_RELEASE(m_pQueue);
```

Comment:
`SAFE_RELEASE` will set its parameter to `NULL` and is only intended when we're destroying/deinitialising and otherwise want to say "we don't want this handle anymore". If the application is AddRef'ing and Release'ing the debug interface then the first Release will break it.

It looks like `m_pQueue` and `m_pList` aren't properly initialised in the parent constructor, so these pointers are NULL - is that the better fix here?